### PR TITLE
Fetch invocation history from clickhouse, merge in unfinished builds from mysql

### PR DESCRIPTION
**Pull request by jdhollen**
**Related issues**: N/A

### CODE REVIEW

**Comment by jdhollen** [comment]
Fully intending to write tests, but hoping to have you all take a look and decide if this is worth it or not.

This only improves performance for queries where the user is either:

1. Not really filtering a ton + using the default sort order (because the range we query from mysql is small), or 
2. Filtering out pending builds (because we won't go to mysql at all)

Short of doing something that lets us skip MySQL entirely, you need some version of this hack.  

If we don't add an index for overall status, then queries for other sort orders will stay slow--they'll require a full scan of all 30 days or whatever the user is looking at (i don't think it's reasonable to add an index for every sort order).

If we add an index and it's still not enough for some orgs, we could make performance better by marking long-disconnected and long-pending builds as "completed" with a janitor task (and flush to clickhouse)--then we could rely the status index to make the MySQL queries for "actually-pending" builds fast, since there shouldn't be *that* many per org at any point in time.

**Review by tylerwilliams — APPROVED** [review]

File: enterprise/server/invocation_search_service/invocation_search_service.go [hunk 1]
```diff
@@ -34,7 +37,8 @@ const (
 )
 
 var (
-	olapInvocationSearchEnabled = flag.Bool("app.olap_invocation_search_enabled", true, "If true, InvocationSearchService will query clickhouse for some queries.")
+	blendedInvocationSearchEnabled = flag.Bool("app.blended_invocation_search_enabled", false, "If true, InvocationSearchService will query clickhouse *and* the regular db.")
+	olapInvocationSearchEnabled    = flag.Bool("app.olap_invocation_search_enabled", true, "If true, InvocationSearchService will query clickhouse for some queries.")
```
Comment (bduffany) [inline enterprise/server/invocation_search_service/invocation_search_service.go:41]: nit: I couldn't figure out the difference between these 2 flags by reading the help strings, maybe the second flag could use more detail (or just have the 1 "blended" flag?)

**Review by bduffany — APPROVED** [review]

File: enterprise/server/invocation_search_service/invocation_search_service.go (hunk 1 above)
Comment (jdhollen) [reply enterprise/server/invocation_search_service/invocation_search_service.go]: Did what I can here to make this clearer (if wordier)

**Review by jdhollen — COMMENTED** [review]
Alright, gonna add tests and submit.

**Pull request merged**